pulp-api/sign-metadata.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+FILE_PATH=$1
+SIGNATURE_PATH="$1.asc"
```

Comment:
```suggestion
SIGNATURE_PATH="$FILE_PATH.asc"
```
Only because there are other changes suggested. :)